docs/diagrams/CustomiseOrderSequenceDiagram.puml

```diff
@@ -1,21 +1,18 @@
 @startuml
 !include style.puml
+scale 1550*500
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":SoConnectParser" as SoConnectParser LOGIC_COLOR
 participant ":CustomiseCommandParser" as CustomiseCommandParser LOGIC_COLOR
 participant ":CustomiseOrderCommandParser" as CustomiseOrderCommandParser LOGIC_COLOR
 participant "u:CustomiseOrderCommand" as CustomiseOrderCommand LOGIC_COLOR
```

Comment:
The instance of CustomiseOrderCommand is named `u` but when returned, it becomes `c`?

Reply:
Ok, nice catch.